doc/src/manual/installation.md

```diff
@@ -0,0 +1,83 @@
+# [Installation](@id man-installation)
+
+There are many ways to install Julia. The following sections highlight the recommended method for each of the main supported platforms, and then present alternative ways that might be useful in specialized situations.
+
+Julia switched the default installation recommendation to a solution based on Juliaup in the summer of 2023. If you installed Julia previously with a method that is _not_ based on Juliaup and want to switch your system to an installation that is based on Juliaup, we recommend that you uninstall all previous Julia versions, ensure that you remove anything Julia related from your `PATH` variable and then install Julia with one of the methods described below.

```

Comment:
Is the information about when this change happened really helpful? Most users wouldn't care, would they? They just want a straight answer "what should I do to install Julia?", I'd reckon (experts who want choices can read on / look elsewhere). So how about this:
```suggestion
The current installation recommendation is a solution based on Juliaup. If you installed Julia previously with a method that is _not_ based on Juliaup and want to switch your system to an installation that is based on Juliaup, we recommend that you uninstall all previous Julia versions, ensure that you remove anything Julia related from your `PATH` variable and then install Julia with one of the methods described below.
```
Though I find it somewhat confusing that it talks about "methods/solutions *based on* Juliaup" instead of just saying "installed using Juliaup". Is there a reason for that? I get that it is technically more accurate, but would the average user care?

(For that matter, I am not starting to wonder: would many users even *know* whether they used Juliaup before? If so, they might read this and wonder "does this apply to me? how do I find out?". I am not making this up, I've had to help dozens of academics installing Julia and it's amazing what all can go wrong and be misunderstood, so much that they also often never upgrade from whatever version we installed -- this is also why I am quite excited about Juliaup, because I am hoping its update capabilities will get more of my colleagues to use recent Julia versions. Anyway, I still might be worrying to much here -- time will tell...)

Reply:
Also, maybe `Juliaup` could be a link to documentation for `Juliaup` (well... I guess that would require that to be created first ;-) but in the meantime perhaps at least a link to https://github.com/JuliaLang/juliaup would be helpful.

Reply:
I've adopted your language for the first recommendation (once I push). I'm still keeping the "based on" because at least for Windows users the term "Juliaup" never shows up when they install, so I'm afraid that a Windows user might be confused by "Installed using Juliaup". But we can iterate, I thought maybe we'll merge this PR rather sooner than later, and then we can refine the language?